src/main/java/io/lighty/yang/validator/formats/JsonTree.java

```diff
@@ -348,8 +390,8 @@ private JSONObject resolveModuleMetadata(final Module module) {
         return jsonModuleMetadata;
     }
 
-    private String resolvePath(final SchemaPath schemaPath) {
-        final Iterable<QName> pathFromRoot = schemaPath.getPathFromRoot();
+    private String resolvePath(final Absolute absolutePath) {
```

Comment:
We can delete this method because Absolute extends SchemaNodeIdentifier.

Reply:
Done